ketcher-autotests/tests/utils/canvas/common/getLeftTopBarSize.ts

```diff
@@ -1,5 +1,5 @@
 import { Page } from '@playwright/test';
-import { getLeftToolBarWidth, getTopToolBarHeight } from '@utils';
+import { getLeftToolBarWidth, getTopToolBarHeight } from '../helpers';
```

Comment:
I believe it's better to use alias. Why did we change it?

Reply:
the imports are done incorrectly, these functions are working only by direct paths to the folders.